### PR TITLE
Update Dockerfile and tools to add conda cleanup

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I started to update the image as in [docker-galaxy-ngs-preprocessing](https://github.com/bgruening/docker-galaxy-ngs-preprocessing/pull/8)

Bérénice

### CODE REVIEW

**Comment by bgruening** [comment]
@bebatut have you cecked that all tools are available from conda already?

**Comment by bebatut** [comment]
Oh I forgot to check :(

File: phylogenetics.yaml [hunk 1]
```diff
@@ -66,14 +65,12 @@ tools:
   tool_panel_section_label: "Phylogeny construction"
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
   install_resolver_dependencies: True
-  install_tool_dependencies: False
```
Comment (bgruening) [inline phylogenetics.yaml]: Since 30 seconds this version should be conda ready :)
https://toolshed.g2.bx.psu.edu/view/rnateam/mafft/15974dd17515

Comment (bebatut) [reply phylogenetics.yaml]: Nice!
`install_tool_dependencies: False` can stay then?

Comment (bgruening) [reply phylogenetics.yaml]: Yes! :)

**Comment by bebatut** [comment]
Any clue about why the tests are failing?

**Comment by bgruening** [comment]
The startup.sh script copies everything into /export with many tools this can take minutes. Increasing the sleep time will help.

**Pull request merged**